forms-flow-web/src/components/Form/List.scss

```diff
@@ -116,24 +107,17 @@ flex-wrap: wrap;
     margin: 10px;
     height: 30px;white-space: nowrap;
 }
-.progress-bar {
-    background-color: #ff0000ab;
-}
-.fileupload-fail {
-    color: #f85757;
-    float: right;
-    font-weight: normal
-}
 
 .progress-bar {
-    background-color: #ff0000ab;
+    background-color: var(--color-red);
```

Comment:
I think we don't need color-red instead of this we can use color-danger or something like that else if some one need to change the color to green he need to update color-red variable with value green

Reply:
color red is renamed with --color-danger in the new commit